### ISSUE TITLE
Key error involving external zone in accessibility calculator

### ISSUE THREAD

**Issue by JoeJimFlood** [IndyMPO/IndyGeoTools#1]
It happens while transit accessibility is being calculated. Here's the error message:

```
Traceback (most recent call last):
  File "P:\MPO\20_Data\IndyGeoprocessingTools\AccessibilityCalculator\AccessibilityCalculator.py", line 252, in <module>
    calc_transit_acc(transit_bool, transit_zones, taz_file)
  File "P:\MPO\20_Data\IndyGeoprocessingTools\AccessibilityCalculator\AccessibilityCalculator.py", line 206, in calc_transit_acc
    emp[:, zone_map[zone[0]]] = zone[1]*bool_skim[:, zone_map[zone[0]]]
KeyError: 154700

Failed to execute (AccessibilityCalculator).
```

**Comment by JoeJimFlood** [comment]
The issue appears to be with all zones, not just zone 154700

**Comment by JoeJimFlood** [comment]
The issue appears to have been caused by cell A1 in the transit skim file reading 'RCIndex' instead of being blank.
